tmk_core/protocol/lufa/adafruit_ble.cpp

```diff
@@ -637,9 +644,7 @@ void adafruit_ble_task(void) {
     if (timer_elapsed(state.last_battery_update) > BatteryUpdateInterval && resp_buf.empty()) {
```

Comment:
If this change works, then I guess the comment above this line should be removed, no?

Reply:
Yep. I've now added that to the pull request.